runtime/integration-tests/src/cases/liquidity_pools_gateway_queue.rs

```diff
@@ -1,17 +1,21 @@
+use cfg_primitives::AccountId;
 use cfg_traits::liquidity_pools::MessageQueue;
 use cfg_types::domain_address::DomainAddress;
 use frame_support::{assert_ok, traits::OriginTrait};
 use pallet_liquidity_pools::Message;
 use pallet_liquidity_pools_gateway::message::GatewayMessage;
+use runtime_common::routing::{AxelarId, RouterId};
+use sp_core::H160;
 use sp_runtime::{traits::One, BoundedVec};
 
 use crate::{
-	cases::liquidity_pools::{DEFAULT_DOMAIN_ADDRESS_MOONBEAM, DEFAULT_ROUTER_ID},
```

Comment:
I wanted to enforce the rule that things should not be imported from one `cases` file to another.

My rationale for doing this:
  - If `cases::liquidity_pools` needs to be refactorized due to some new feature, we're probably breaking also the queue tests (when we shouldn't).
  - If a reader wants to understand these `queue` tests, we force them to also look into `liquidity_pools` tests to fully understand when both are theoretically unrelated.

Of course, if the common code grows, we should factorize this into some well-through enough-generic utility module. Then, both cases can point to those logic without carrying mutual information.

cc @cdamian @wischli

Reply:
Thanks for bringin this up. I definitely agree!